portfolio/src/main/java/com/google/sps/data/UserLogin.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+/**
+* User Login class to hold fields relating to the user login process
+*/
+public class UserLogin {
```

Comment:
Good idea using an object to represent this!

Reply:
thanks!